tests/rkt_non_root_test.go

```diff
@@ -25,6 +25,14 @@ import (
 
 // TestNonRootReadInfo tests that non-root users that can do rkt list, rkt image list.
 func TestNonRootReadInfo(t *testing.T) {
+	if !common.SupportsUserNS() {
```

Comment:
The test starts some pods: some with user namespaces and some without. Then it checks that `rkt list` works without being root.

I think the idea was to check that non-root access to `/var/lib/rkt/pods/run/$UUID/stage1/rootfs/rkt/status/` works fine, even when the directory uid/gid is shifted by the user namespace. The test also tries with and without overlayfs because the "rkt/status" directory would be stored in a different location in case of overlay fs.

However, the test only runs `rkt list` and not `rkt status`. And I don't think `rkt list` needs to have access to the "rkt/status" directory...